src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
1. is an ok hack, although it will result in a bunch of duplicate files being dropped into the root Tools directory (like coreclr, clrjit, S.P.CL, etc.) which is not great (at least IMO). I think we can exclude them from being copied out (with some metadata in the project.json?), but then we need to post-copy ilasm down into the shared folder...